src/Apps/Order/Routes/__tests__/Offer.test.tsx

```diff
@@ -0,0 +1,72 @@
+import { Input } from "Components/Input"
+import { renderRelayTree } from "DevTools"
+import React from "react"
+import { graphql } from "react-relay"
+import { UntouchedOrder } from "../../../__test__/Fixtures/Order"
+import { TransactionSummary } from "../../Components/TransactionSummary"
+import { OfferFragmentContainer as OfferRoute } from "../Offer"
+
+jest.unmock("react-relay")
+
+describe("Payment", () => {
+  beforeEach(() => {
+    console.error = jest.fn() // Silences component logging.
+  })
+
+  const defaultProps = {
+    router: {
+      push: jest.fn(),
+    } as any,
+    route: {
+      onTransition: jest.fn(),
+    } as any,
+    relay: {
+      environment: {},
+    } as any,
+    mediator: {
+      trigger: jest.fn(),
+    } as any,
+  }
+
+  const getWrapper = () => {
+    return renderRelayTree({
+      Component: ({ order }: any) => (
+        <OfferRoute order={order} {...defaultProps} />
+      ),
+      query: graphql`
+        query OfferTestQuery {
+          order: ecommerceOrder(id: "unused") {
+            ...Offer_order
+          }
+        }
+      `,
+      mockResolvers: {
+        Order: () => UntouchedOrder,
+      },
+      renderUntil: wrapper => wrapper.find(Input).length > 0,
+    })
+  }
+
+  it("renders", async () => {
+    const wrapper = await getWrapper()
```

Comment:
If `getWrapper` doesn't accept params might be better to move to 

```js
beforeAll(async () => { 
  wrapper = await getWrapper()
})
``` 
so that it can be used without needing `async / await` on every function.

Reply:
good idea!